[2024-04-13 20:54:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:21] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 2680fb919c17c10a [::1]:49794 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:54:21] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 2680fb919c17c10a [::1]:20132 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:54:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 20:54:54] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 2680fb919c17c10a [::1]:39976 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:54:54] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 2680fb919c17c10a [::1]:20132 failed checkpoint posthandshake: Connected to self
[2024-04-13 20:54:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
